jemalloc-prof.spec

```diff
@@ -0,0 +1,25 @@
+%define jemalloc_version 5.2.1
```

Comment:
@gartung why not use `## INCLUDE jemalloc-common` just like jemalloc-debug https://github.com/cms-sw/cmsdist/blob/IB/CMSSW_12_5_X/master/jemalloc-debug.spec#L1-L2 ?

Reply:
I needed to add `Requires: libunwind`.

Reply:
can you do that by adding `Requires: libunwind` before the `## INCLUDE jemalloc-common` line e.g.
```
Requires: libunwind
## INCLUDE jemalloc-common
```
should work

Reply:
Yes that worked. Now I need to figure out why libunwind is not linked to libjemalloc.